tests/molbe_octane_test.py

```diff
@@ -17,7 +20,58 @@
     reason="This test is known to fail.",
 )
 def test_octane_molbe():
-    # Perform pyscf HF calculation to get mol & mf objects
+    # Prepare octane molecule
+    mol, mf = prepare_octane()
+
+    # Perform CCSD calculation to get reference energy for comparison
+    mc = cc.CCSD(mf, frozen=8)
+    mc.verbose = 0
+    ccsd_ecorr = mc.kernel()[0]
+    print(f"*** CCSD Correlation Energy: {ccsd_ecorr:>14.8f} Ha", flush=True)
+
+    # initialize fragments (use frozen core approximation)
+    fobj = fragpart(be_type="be2", mol=mol, frozen_core=True)
+    # Initialize BE
+    mybe = BE(mf, fobj)
+
+    # Perform BE density matching.
+    # Uses 20 procs, each fragment calculation assigned OMP_NUM_THREADS to 4
+    # effectively running 5 fragment calculations in parallel
+    mybe.optimize(solver="CCSD", nproc=20, ompnum=4)
+
+    # Compute error
+    be_ecorr = mybe.ebe_tot - mybe.ebe_hf
+    err_ = (ccsd_ecorr - be_ecorr) * 100.0 / ccsd_ecorr
```

Comment:
I know the test should something else, but why not add an assert here for the energy.
It's a cheap thing.

Reply:
This one is the existing octane test that we have not activated yet because of the technical issues. It shows as 'changed' in diff because I factored out the repeated `mol, mf` part to a separate function. We should maybe address this when we finally activate this test.